docling/backend/hwp_backend.py

```
@@ -32,69 +30,64 @@ def __init__(self, in_doc: InputDocument, path_or_stream: Union[Path, BytesIO])
-            if not os.path.exists(output_hwpx):
-                raise RuntimeError(f"HWPX file was not created: {output_hwpx}")
-            
-            return Path(output_hwpx)
-            
-        except Exception as e:
-            raise RuntimeError(f"Failed to convert HWP to HWPX: {e}")
+        """hwp2hwpx.sh 스크립트로 HWP → HWPX 변환."""
+        input_hwp = str(hwp_path)
+        output_hwpx = str(hwp_path.with_suffix('.hwpx'))
+
+        result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)